linux_os/guide/system/bootloader-grub2/grub2_password/rule.yml

```diff
@@ -8,28 +8,15 @@ description: |-
     The grub2 boot loader should have a superuser account and password
     protection enabled to protect boot-time settings.
     <br /><br />
-    To do so, select a superuser account name and password and and modify the
-    <tt>/etc/grub.d/01_users</tt> configuration file with the new account name.
-    <br /><br />
     Since plaintext passwords are a security risk, generate a hash for the pasword
     by running the following command:
     <pre>$ grub2-setpassword</pre>
     When prompted, enter the password that was selected.
     <br /><br />
-    NOTE: It is recommended not to use common administrator account names like root,
-    admin, or administrator for the grub2 superuser account.
-    <br /><br />
-    Change the superuser to a different username (The default is 'root').
-    <pre>$ sed -i s/root/bootuser/g /etc/grub.d/01_users</pre>
-    <br /><br />
-    To meet FISMA Moderate, the bootloader superuser account and password MUST
-    differ from the root account and password.
-    Once the superuser account and password have been added,
```

Comment:
Why would this text be removed? Is it no longer accurate?

Reply:
The same case as text commented below.